### ISSUE TITLE
Resizing terminal window throws Segmentation fault

### ISSUE THREAD

**Issue by simonlovesyou** [anseki/readline-sync#24]
If you resize the terminal window when prompting the user for input it throws 'Segmentation fault: 11'

**Comment by anseki** [comment]
Hi simonlovesyou, thank you for your post.

So, what is your terminal? and your environment?
e.g. LXTerminal 0.1.8 on Ubuntu 15.10, running NodeJS 4.2.3.

And, can you show me your code?

**Comment by simonlovesyou** [comment]
Hi anseki,

Terminal.app Version 2.6.1. OS X 10.11.12, running NodeJS v.5.0.0

This is the only code I need to reproduce it:
``
'use strict';
let prompt = require('readline-sync');
prompt.question("Waiting for user input...\n");
``
If I resize the Terminal window while the application is waiting for user input I get 
``
Segmentation fault: 11
``
and the application shuts down.

It might be a problem with the Terminal application? Haven't tried with any other console/terminal application, could do that later.

**Comment by anseki** [comment]
Thank you for the information.

It seems that a bug exists in NodeJS (iojs) v3.3.0+.
I think that libuv-v1.7.3+ which is included by iojs-v3.3.0 has this bug.
I will report this bug to NodeJS team later.

If you want to avoid this bug, you can use this:

```js
let readlineSync = require('readline-sync');
readlineSync._DBG_set_useExt(true);
```

`_DBG_set_useExt` method that is given `true` makes readlineSync run without libuv.
But, I don't recommend this, because this makes your script be slow, and users usually don't resize a terminal window, maybe.
And I think that NodeJS team fixes this bug soon.

BTW, your code output a message "Waiting for user input...", if you want to make the script pause, you can use [`keyInPause` method](https://github.com/anseki/readline-sync#utility_methods-keyinpause).

**Comment by anseki** [comment]
NodeJS was fixed via 5994
